Sources/UI/Chat/Attachments/AttachmentPreviewProtocol.swift

```diff
@@ -11,9 +11,10 @@ import StreamChatClient
 import StreamChatCore
 import RxSwift
 
-protocol AttachmentPreviewProtocol where Self: UIView {
+protocol AttachmentPreview where Self: UIView {
```

Comment:
Isn't this the same as the file above?
`protocol AttachmentPreview where Self: UIView`

Reply:
Yes, I renamed `AttachmentPreviewProtocol` to `AttachmentPreview`.